Makefile

```
@@ -102,6 +102,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac\
 		webhook output:webhook:artifacts:config=config/components/webhook\
 		paths="./pkg/..."
+	cp ./config/components/crd/bases/jobset.x-k8s.io_jobsets.yaml charts/jobset/crds/.
```

I think, we should do force copy: https://github.com/kubeflow/trainer/blob/master/Makefile#L122C2-L123C1

```suggestion
	cp -f ./config/components/crd/bases/jobset.x-k8s.io_jobsets.yaml ./charts/jobset/crds/
```